### PR TITLE
chore(mailtools): fix and add more tests to mailtools

### PR DESCRIPTION
**Pull request by BlankParticle**
## What does this PR do?

Fixes existing fixture tests and add more tests.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [x] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [x] Self-reviewed my own code
- [x] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

File: packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html [hunk 1]
```diff
@@ -15,11 +15,11 @@
...
     <p>Kind Regards</p>
     <p>Henrique</p>
-    <p>--<br />I use <a href="https://www.yourtempo.co">Tempo</a> to improve my focus</p>
+    <p>--<br />I use <a href="https://www.yourtempo.co" title="https://www.yourtempo.co">Tempo</a> to improve my focus</p>
     <br /><br />
     <blockquote class="gmail_quote" style="margin:0">
       On Tue, Apr 13 2021 at 10:37 AM (GMT+2),
-      <a href="mailto:mette.randers@dk.gt.com" target="_blank" rel="noopener noreferrer">mette.randers@dk.gt.com</a>
+      <a href="mailto:mette.randers@dk.gt.com" target="_blank" rel="noopener noreferrer" title="mailto:mette.randers@dk.gt.com">mette.randers@dk.gt.com</a>
       wrote: <br /><br />
```
Comment (McPizza0) [inline packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html:23]: lets remove this email address from the test - for privacy

File: packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html [hunk 2]
```diff
@@ -136,7 +136,7 @@
                   lang="EN-US"
                   style='font-size:9.0pt;font-family:"Arial",sans-serif;color:black;mso-fareast-language:DA'
                 >
-                  <a href="tel:+4535275268" target="_blank" rel="noopener noreferrer">+4535275268</a></span
+                  <a href="tel:+4535275268" target="_blank" rel="noopener noreferrer" title="tel:+4535275268">+4535275268</a></span
```
Comment (McPizza0) [inline packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html:139]: remove value for privacy

File: packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html [hunk 3]
```diff
@@ -148,23 +148,23 @@
                   lang="EN-US"
                   style='font-size:9.0pt;font-family:"Arial",sans-serif;color:black;mso-fareast-language:DA'
                 >
-                  <a href="tel:+4533110220" target="_blank" rel="noopener noreferrer">+4533110220</a></span
+                  <a href="tel:+4533110220" target="_blank" rel="noopener noreferrer"  title="tel:+4533110220">+4533110220</a></span
```
Comment (McPizza0) [inline packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html:151]: remove value for privacy

File: packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html [hunk 4]
```diff
@@ -148,23 +148,23 @@
...
-                  >&#xA0; </span
+                  >&nbsp; </span
                 ><span lang="EN-US" style="mso-fareast-language:DA"><br /> </span
                 ><b
                   ><span lang="EN-US" style='font-size:9.0pt;font-family:"Arial",sans-serif;mso-fareast-language:DA'
                     >M</span
                   ></b
                 ><span lang="EN-US" style='font-size:9.0pt;font-family:"Arial",sans-serif;mso-fareast-language:DA'>
-                  <a href="tel:+4526881988" target="_blank" rel="noopener noreferrer">+4526881988</a><b> </b></span
+                  <a href="tel:+4526881988" target="_blank" rel="noopener noreferrer" title="tel:+4526881988">+4526881988</a><b> </b></span
```
Comment (McPizza0) [inline packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html:160]: remove value for privacy

File: packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html [hunk 5]
```diff
@@ -148,23 +148,23 @@
...
                 ><span lang="EN-US" style="mso-fareast-language:DA"><br /> </span
                 ><b
                   ><span lang="EN-US" style='font-size:9.0pt;font-family:"Arial",sans-serif;mso-fareast-language:DA'
-                    >E&#xA0;</span
+                    >E&nbsp;</span
                   ></b
                 ><span style='font-size:9.0pt;font-family:"Arial",sans-serif;color:black;mso-fareast-language:DA'
-                  ><a href="mailto:Mette.Randers@dk.gt.com"
+                  ><a href="mailto:Mette.Randers@dk.gt.com"  title="mailto:Mette.Randers@dk.gt.com"
```
Comment (McPizza0) [inline packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html:167]: remove value for privacy

File: packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html [hunk 6]
```diff
@@ -148,23 +148,23 @@
...
                     ><span lang="EN-US" style="color:blue">Mette.Randers@dk.gt.com</span></a
```
Comment (McPizza0) [inline packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html:168]: remove value for privacy

File: packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html [hunk 7]
```diff
@@ -188,8 +188,9 @@
                       style="width:.15in;height:.15in"
                       id="_x0000_i1027"
                       src="cid:image001.png@01D73051.10303D00"/></span></a
-                ><span lang="EN-US" style="mso-fareast-language:DA">&#xA0;|&#xA0;</span
+                ><span lang="EN-US" style="mso-fareast-language:DA">&nbsp;|&nbsp;</span
                 ><a href="https://teams.microsoft.com/l/chat/0/0?users=Mette.Randers@dk.gt.com&amp;topicname=Chat"
+                title="https://teams.microsoft.com/l/chat/0/0?users=Mette.Randers@dk.gt.com&amp;topicname=Chat"
```
Comment (McPizza0) [inline packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html:193]: remove value for privacy

File: packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html [hunk 8]
```diff
@@ -200,8 +201,8 @@
                       src="cid:image002.png@01D73051.10303D00"/></span></a
                 ><b
                   ><span style="font-size:9.0pt;mso-fareast-language:DA"
-                    ><a href="https://teams.microsoft.com/l/chat/0/0?users=Mette.Randers@dk.gt.com&amp;topicname=Chat"
-                      ><span lang="EN-US" style="color:#3399FF">&#xA0;Chat with me on Teams!</span></a
+                    ><a href="https://teams.microsoft.com/l/chat/0/0?users=Mette.Randers@dk.gt.com&amp;topicname=Chat" title="https://teams.microsoft.com/l/chat/0/0?users=Mette.Randers@dk.gt.com&amp;topicname=Chat"
+                      ><span lang="EN-US" style="color:#3399FF">&nbsp;Chat with me on Teams!</span></a
                     ></span
```
Comment (McPizza0) [inline packages/mailtools/src/tests/prepareMessage/fixtures/trimmed-reply-bug.output-complete.html:206]: remove value for privacy

**Review by McPizza0 — CHANGES_REQUESTED** [review]

**Comment by McPizza0** [comment]
just a few values to remove

**Comment by BlankParticle** [comment]
Done!

**Review by McPizza0 — APPROVED** [review]

**Pull request merged**